daemon/api.go

```diff
@@ -1193,6 +1187,18 @@ func (inst *snapInstruction) errToResponse(err error) Response {
 		default:
 			return InternalError("store.SnapNotFound with %d snaps", len(inst.Snaps))
 		}
+	case store.ErrRevisionNotAvailable:
+		switch len(inst.Snaps) {
+		case 1:
+			return SnapRevisionNotAvailable(inst.Snaps[0], err)
+		// store.ErrRevisionNotAvailable should only be
```

Comment:
I think this comment would make more sense when placed directly below `case store.ErrRevisionNotAvailable`.